src/pages/RegisterPage/index.js

```diff
@@ -15,169 +15,158 @@ const RegisterPagePage = () => {
   const [username, setUsername] = useState("");
   const [country, setCountry] = useState("");
 
-  useEffect(() => {
-    if (supabaseClient.auth.user())
-      navigate("/listingspage");
-  });
-
   return (
     <div className={registerPageStyles["parent"]}>
-      
-        <div className={registerPageStyles["form"]}>
-          <img
-            src={"/images/img_minervaLogo.png"}
-            className={
-              registerPageStyles["minerva_logo_1-removebg-preview_1-2"]
-            }
-            onClick={handleNavigate21}
-            alt="minerva"
-          />
-          <div className={`${registerPageStyles["username-input"]}`}>
-            <UserInput input1="Username" />
-            <div
-              className={`${registerPageStyles["input-box-set-master"]} border-1px-mischka`}
-            >
-              <div className={registerPageStyles["input-text"]}>
-                <div
-                  className={`${registerPageStyles["text"]} nunitosans-normal-storm-gray-28px`}
-                >
-                  <Input
-                    type="text"
-                    value={username}
-                    onChange={(e) => setUsername(e.target.value)}
-                    placeholder="Enter username"
-                  ></Input>
-                </div>
-              </div>
-            </div>
-          </div>
-
-          <div className={`${registerPageStyles["username-input"]}`}>
-            <UserInput input1="Email" />
-            <div
-              className={`${registerPageStyles["input-box-set-master"]} border-1px-mischka`}
-            >
-              <div className={registerPageStyles["input-text"]}>
-                <div
-                  className={`${registerPageStyles["text"]} nunitosans-normal-storm-gray-28px`}
-                >
-                  <Input
-                    type="email"
-                    value={email}
-                    onChange={(e) => setEmail(e.target.value)}
-                    placeholder="Enter email"
-                  ></Input>
-                </div>
+      <div className={registerPageStyles["form"]}>
+        <img
+          src={"/images/img_minervaLogo.png"}
+          className={registerPageStyles["minerva_logo_1-removebg-preview_1-2"]}
+          onClick={handleNavigate21}
```

Comment:
Can just replace `handleNagivate21` with `navigate('/')` since `handleNagivate21` is only used once in the entire file ==> don't need to declare a constant for it

Reply:
Alternatively, can change `handleNavigate21` to a clearer name such as `navigateHome` if you wish to keep it as a constant to make things neater